week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,13 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidy(name) {
+  return name.trim().toLowerCase().replace("/", "");
+}
+ function tidyUpString(strArr) {
+  var tidystr = strArr.map(tidy);
+ return tidystr
+ }
```

Comment:
Sulaiman actually you are passing an array as a parameter. You can go through an array by using for loop. Map is used to create a new array with some modification of the first array. You can either use for loop or map to solve this exercise. Try to do it again.

Reply:
done